app/scripts/modules/core/src/notification/selector/types/githubstatus/GithubNotificationType.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { FormikFormField, TextInput, Validators } from 'core/presentation';
+import { INotificationTypeCustomConfig } from 'core/domain';
+
+export class GithubNotificationType extends React.Component<INotificationTypeCustomConfig> {
+  public render() {
+    const { fieldName } = this.props;
+    return (
+      <>
+        <div className="sp-margin-m-bottom">
+          <div className={'form-group'}>
+            <label className={'col-md-4 sm-label-right'}>Github Repository</label>
```

Comment:
I *think* the label can be included as a prop of `FormikFormField` but going to defer to @Jammy-Louie for the convention here since he very kindly wrote many similar components in React.

Reply:
You are right you can include it as a `label` prop and it is recommended to do so. However it causes misalignment in the `manualExecution` modal when it is rendered.

Going to have to go back and realign everything in notifications using the `layout` prop (to do item) or realign everything in the `manualExecution` modal.

TLDR; This is fine for now..

Reply:
updated the PR to have everything align properly with `label` as a props of `FormikFormField`